tests/common_fixtures.py

```diff
@@ -485,6 +486,22 @@ def _get_function_internal(self, function_name: str = None):
     def store_metric(self, uid, project="", keyvals=None, timestamp=None, labels=None):
         pass
 
+    def list_hub_sources(self, item_name: str = None, tag: str = None):
```

Comment:
Is this used?

Reply:
No, removing

Reply:
In the end I am using it - added `rundb_mock` in `test_helpers.py`